model/context.go

```diff
@@ -125,7 +125,7 @@ func DecodeContext(input interface{}, cfg Config, err error) (*Context, error) {
 	}
 
 	decoder := utility.ManualDecoder{}
-	field := fields.Mapper(cfg.HasShortFieldNames)
+	field := field.Mapper(cfg.HasShortFieldNames)
```

Comment:
The var `field` is shadowing the package name now. How about changing the variable to `mapper`?

Reply:
Yeah... amended